packages/web/integrations/axelar/types.ts

```diff
@@ -61,4 +62,6 @@ export type SourceChainConfig = {
   };
 
   logoUrl: string;
+  
```

Comment:
I've copied down the comment this transferMinFee property had above. Should be sufficient description to avoid the potential confusion you've pointed out.